src/services/search/searchResult.test.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @jest-environment node
+ */
+import { SearchEngine } from '@prisma/client';
+
+import request from '@/lib/searchRequest';
+
+import { searchResult } from './searchResult';
+
+jest.mock('../../lib/searchRequest.ts');
+const mockedAxios = request as jest.Mocked<typeof request>;
+
+describe('searchResult', () => {
+  describe('given axios call resolves data', () => {
```

Comment:
Axios could be left out of the test description:

```suggestion
  describe('given a succesful request to fetch the search result page', () => {
```